soc/posix/inf_clock/soc.h

```diff
@@ -7,7 +7,6 @@
 #ifndef _POSIX_SOC_INF_CLOCK_SOC_H
 #define _POSIX_SOC_INF_CLOCK_SOC_H
 
-#include "board_soc.h"
```

Comment:
This include must remain here. It is one of the basic interfaces in the posix arch.. This header allows a "board" to expose provide/override SOC definitions.